### ISSUE TITLE
When there is only one tab, it gets center aliened in this widget. 

### ISSUE THREAD

**Issue by ssharanyab** [railson-ferreira/scrollable_list_tab_scroller#5]

**Comment by railson-ferreira** [comment]
That is the default behavior. There is an option in case you don't want this behavior:

```
headerContainerBuilder: (context, child) {
          return SizedBox(
            width: double.maxFinite,
            height: 30, // Default header height
            child: child,
          );
        }
```

**Comment by railson-ferreira** [comment]
It seems that the issue has been resolved based on the information provided. However, since there hasn't been any confirmation from the reporter, I'm closing this issue now. If you encounter the problem again or have further questions, please feel free to reopen it.

Closes #5

**Comment by railson-ferreira** [comment]
It seems that the issue has been resolved based on the information provided. However, since there hasn't been any confirmation from the reporter, I'm closing this issue now. If you encounter the problem again or have further questions, please feel free to reopen it.

Closes #5
